examples/csharp/Dockerfile

```diff
@@ -20,17 +22,10 @@ RUN rm -rf src/Twilio/Rest/*/
 COPY src/Twilio/Rest src/Twilio/Rest
 
 RUN rm -rf test/Twilio.Test/*/
-COPY unit-test/Rest integrationTest/Rest test/Twilio.Test/
+COPY unit-test/Rest test/Twilio.Test/
 
 RUN dotnet restore
 RUN dotnet build -c Release
-#RUN dotnet test test/Twilio.Test/Twilio.Test.csproj --no-build
-# sonar coverage reports
-# Add /d:sonar.login="${SONAR_TOKEN}" to sync up with sonarcloud.
-RUN dotnet sonarscanner begin /k:"twilio_twilio-csharp" /o:"twilio" /d:sonar.host.url=https://sonarcloud.io /d:sonar.language="cs" /d:sonar.exclusions="src/Twilio/Rest/**/*.*,test/Twilio.Test/**/*.*" /d:sonar.cs.opencover.reportsPaths="test/lcov.net451.opencover.xml"
```

Comment:
There's an issue here in that twilio-csharp explicitly ignores the `rest` folder, but in this repo we want to cover it.

Reply:
Should i change this to a variable and pass different values from twiliio-oai-generator repo and twilio-csharp repo for their respective runs?

Reply:
Altered make commands to accept sonar exclusions as a variable and from csharp repo run, we are now passing rest folder as well, and from twilio-oai-generator repo only test folder is excluded